docs/cloud-integration.md

```diff
@@ -254,8 +254,8 @@ Here is the documentation on the standard connectors both from Apache and the cl
 * [Azure Data Lake Gen 1](https://hadoop.apache.org/docs/current/hadoop-azure-datalake/index.html).
 * [Hadoop-AWS module (Hadoop 3.x)](https://hadoop.apache.org/docs/current3/hadoop-aws/tools/hadoop-aws/index.html).
 * [Amazon S3 via S3A and S3N (Hadoop 2.x)](https://hadoop.apache.org/docs/current/hadoop-aws/tools/hadoop-aws/index.html).
```

Comment:
Could you please update this line, too? This is because s3n connector has been removed.

Reply:
I have updated link to reference Hadoop 2, which still supports S3N even though it's deprecated. Let me know if it's better to remove reference to S3N.

Reply:
Thank you for updating the link. It is fine with me.